client/my-sites/domains/domain-management/README.md

```diff
@@ -1,14 +1,13 @@
-Domain Management
-=================
+# Domain Management
 
 This folder contains components specific to the `/domains/manage` routes.
 
 Supported routes:
 
 - `/domains/manage` - entry point for domains management
 - `/domains/manage/:site` - lists domains for a site
-- `/domains/manage/:site/add-google-apps` - adds Google Apps for a site
-- `/domains/manage/:domain/add-google-apps/:site` - adds Google Apps for a domain
+- `/domains/manage/:site/add-gsuite-users` - adds G Suite for a site
+- `/domains/manage/:domain/add-gsuite-users/:site` - adds G Suite for a domain
```

Comment:
@belcherj: it might be a good idea to update Happiness Engineers on that change since they might still direct users to the former url.